contracts/price-feed-mock/src/lib.rs

```diff
@@ -0,0 +1,41 @@
+#![deny(warnings)]
+#![no_std]
+
+use price_feed_interface::{PriceData, PriceFeedTrait};
+use soroban_sdk::{contractimpl, Address, Env, Vec};
+
+pub struct PriceFeedMock;
+
+#[contractimpl]
+impl PriceFeedTrait for PriceFeedMock {
+    fn base(_env: Env) -> Address {
+        todo!()
```

Comment:
I think it's better use `unimplemented!()` macro in such cases
```suggestion
        todo!()
```